### DISCUSSION TITLE
OSU! Lazer crashing as soon as i click

### DISCUSSION THREAD

**Discussion by EthanMCT** [ppy/osu#18134] [Q&A]
Very weird bug happening when i open Lazer, the game opens on but two errors come up, shown in the screenshot below, as soon as i click anything on the game window it freezes and crashes, everything i have tried has not worked, i even completely reinstalled lazer and it did nothing.

Lazer used to work for me a short while ago but in the last week it has been completely broken.

[performance.log](https://github.com/ppy/osu/files/8645125/performance.log)
[runtime.log](https://github.com/ppy/osu/files/8645126/runtime.log)
[updater.log](https://github.com/ppy/osu/files/8645127/updater.log)
[database.log](https://github.com/ppy/osu/files/8645128/database.log)
[network.log](https://github.com/ppy/osu/files/8645129/network.log)

Update: so it kind of works when i unplug my tablet, it worked once but as soon as i plug it back in it stops working

now it has gone back to not working even if tablet is unplugged

i own a CTL 472 and use default wacom drivers

**Comment by frenzibyte** [comment]
![CleanShot 2022-05-07 at 13 34 10@2x](https://user-images.githubusercontent.com/22781491/167250662-bf57578a-2a26-4e8b-9076-0316b75da86c.png)

https://github.com/ppy/osu/discussions/15188

**Reply by frenzibyte** [reply]
In addition, the two errors you're getting are related to the tablet you have connected (https://github.com/ppy/osu/issues/13461)

**Reply by EthanMCT** [reply]
how do i get to my logs?

**Reply by integerrr** [reply]
Just read #15188, please

**Reply by EthanMCT** [reply]
added them, my bad

**Comment by frenzibyte** [comment]
Attached logs show that you've already entered the game one hour after posting this, and show no sign of a crash at that point.

**Reply by EthanMCT** [reply]
yea i meant to update it, worked when i unplugged my tablet so idk if its a problem with drivers

**Reply by frenzibyte** [reply]
Interesting, can you plug back your tablet and attach logs after getting the crash? that would ascertain whether this is an issue we can handle framework-side.

**Reply by EthanMCT** [reply]
here

[database.log](https://github.com/ppy/osu/files/8645334/database.log)
[network.log](https://github.com/ppy/osu/files/8645335/network.log)
[performance.log](https://github.com/ppy/osu/files/8645336/performance.log)
[runtime.log](https://github.com/ppy/osu/files/8645337/runtime.log)
[updater.log](https://github.com/ppy/osu/files/8645338/updater.log)

**Reply by peppy** [reply]
None of these logs show any crashes. I also can't seem to find the screenshots you mention were attached to the opening post.
